_commands/basics/df.md

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+df
+--
+
+df displays the amount of disk space available on the file system containing each file name argument. If no file name is given, the space available on all currently mounted file systems is shown. Disk space is shown in 1K blocks by default, unless the environment variable POSIXLY_CORRECT is set, in which case 512-byte blocks are used.  
```

Comment:
This is straight out of the manpage – the goal for the reference pages is to be a little friendlier. Could you trim this down to only a sentence or two about `df` that's more elementary?